Sources/StatsdClient/Atomics.swift

```diff
@@ -61,6 +61,62 @@ internal class AtomicCounter {
         self.nio.store(value)
         #endif
     }
+
+    func wrappingIncrementThenLoad() -> Int64 {
+        #if canImport(Atomics)
+        self.managed.wrappingIncrementThenLoad(ordering: .sequentiallyConsistent)
+        #else
+        let value = self.nio.add(1)
+        return value + 1
```

Comment:
this is not ideal and I plan to remove support to older version than 5.6 in a follow up PR so that we can remove the NIO atomics related code